### PR TITLE
feat : created initial version of a README

### PR DESCRIPTION
**Pull request by rel1c-hub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,131 @@
-read me
+b# Book Store Project
```
Comment (yevheniisynytsia) [inline README.md]:
```suggestion
# Book Store Project
```

File: README.md [hunk 2]
```diff
@@ -1 +1,131 @@
...
+    MYSQLDB_DOCKER_PORT=3306  
+    SPRING_LOCAL_PORT=8088  
+    SPRING_DOCKER_PORT=8080  
+    DEBUG_PORT=5005
+    ```
+4. **Build Project**: Run `mvn clean package`.
+5. **Build Docker Container**: Use `docker-compose build`.
+6. **Run Docker Container**: Use `docker-compose up`.
+7. **Access Application**: Visit `http://localhost:8088/api` and test using Postman/Swagger.
+    - For admin features, use credentials:
```
Comment (yevheniisynytsia) [inline README.md]: let's say that a client can use either basic auth, or jwt bearer token

File: README.md [hunk 3]
```diff
@@ -1 +1,131 @@
...
+        "email": "harrypotter@wp.pl",
+        "password": "12345678"
+      }
+      ```
+8. **Stop and Remove Containers**: Use `docker-compose down`.
+
+## Possible Improvement
+
+- **Payment Integration**: Implement a payment system using the Stripe API to facilitate secure online payments, enhancing user convenience.
+
```
Comment (yevheniisynytsia) [inline README.md]: let's add one more thing - for example author model with detailed info or similar

File: README.md [hunk 4]
```diff
@@ -1 +1,131 @@
...
+    - For user features, use credentials:
+      ```json
+      {
+        "email": "harrypotter@wp.pl",
+        "password": "12345678"
+      }
+      ```
+8. **Stop and Remove Containers**: Use `docker-compose down`.
+
+## Possible Improvement
```
Comment (yevheniisynytsia) [inline README.md:147]: I would also like to see some part about the challenges you've faced and how you overcame those

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1 +1,155 @@
...
+- 🟩 **POST** `/auth/login`: User login.
+
+
+### Book
+- 🟨 **GET** `/books`: Retrieve book catalogue.
+- 🟨 **GET** `/books/{id}`: View book details.
+- 🟨 **GET** `/books/search`: Search books.
+- 🟥 **POST** `/books`: Add a new book.
+- 🟥 **PUT** `/books/{id}`: Update book details.
+- 🟥 **DELETE** `/books/{id}`: Delete book.
```
Comment (karma-o) [inline README.md:76]: I do not understand why some requests are red and some are yellow, so please add a legend(explanation) of the color-coding at the top of that topic

File: README.md [hunk 6]
```diff
@@ -1 +1,155 @@
...
+- 🟨 **GET** `/orders/{id}/items`: View order items.
+- 🟨 **GET** `/orders/{orderId}/items/{itemId}`: View specific order item.
+- 🟨 **POST** `/orders`: Place order.
+- 🟥 **PATCH** `/orders/{id}`: Update order status.
+
+
+## SQL Database Diagram
+
+
+![Database Diagram]
```
Comment (karma-o) [inline README.md:106]: you should push the image to the git as well(some "readme resources" folder in the root of the project) to be able to see it on the GitHub and not only on your local env

File: README.md [hunk 7]
```diff
@@ -1 +1,155 @@
...
+  ```plaintext
+  MYSQLDB_USER=root 
+   MYSQLDB_ROOT_PASSWORD=root 
+   MYSQLDB_DATABASE=test_db 
+   MYSQLDB_LOCAL_PORT=3307 
+   MYSQLDB_DOCKER_PORT=3306 
+   SPRING_LOCAL_PORT=8088 
+   SPRING_DOCKER_PORT=8080 
+   DEBUG_PORT=5005
+   ```
```
Comment (karma-o) [inline README.md:124]: redundant spaces and wrong file type
```suggestion
  ```env
  MYSQLDB_USER=root 
  MYSQLDB_ROOT_PASSWORD=root 
  MYSQLDB_DATABASE=test_db 
  MYSQLDB_LOCAL_PORT=3307 
  MYSQLDB_DOCKER_PORT=3306 
  SPRING_LOCAL_PORT=8088 
  SPRING_DOCKER_PORT=8080 
  DEBUG_PORT=5005
  ```
```

File: README.md (hunk 4 above)
Comment (karma-o) [reply README.md:147]: agree, would be nice to have it

**Review by karma-o — CHANGES_REQUESTED** [review]
Looking good but still have to make some fixes

**Pull request merged**